tests/smartswitch/common/device_utils_dpu.py

```diff
@@ -23,7 +23,7 @@ def num_dpu_modules(platform_api_conn):
     return num_modules
 
 
-@pytest.fixture(scope='function')
+@pytest.fixture(scope='function', autouse=True)
```

Comment:
Can we change the scope to module?  so, it only run one time for the tests in one module

Reply:
Making it as module scope resulted in error as follows.
It tries to use platform_api_conn (another fixture) which is of functional scope.

Example:
ERROR smartswitch/platform_tests/test_show_platform_dpu.py::test_midplane_ip[str2-7050cx3-acs-03] - Failed: ScopeMismatch: You tried to access the function scoped fixture platform_api_conn with a module scoped request object, involved factories:

Reply:
Got it. Thanks